service/dap/server_test.go

```diff
@@ -1462,17 +1462,12 @@ func TestEvaluateCallRequest(t *testing.T) {
 					// and not the return to the original point of injection after it
 					// completed.
 					client.EvaluateRequest("call makeclos(nil)", 1000, "not watch")
-					client.ExpectStoppedEvent(t)
+					stopped := client.ExpectStoppedEvent(t)
 					erres = client.ExpectVisibleErrorResponse(t)
 					if erres.Body.Error.Format != "Unable to evaluate expression: call stopped" {
 						t.Errorf("\ngot %#v\nwant Format=\"Unable to evaluate expression: call stopped\"", erres)
 					}
-					if (goversion.VersionAfterOrEqual(runtime.Version(), 1, 15) && (runtime.GOOS == "linux" || runtime.GOOS == "windows")) ||
-						runtime.GOOS == "freebsd" {
```

Comment:
The goroutine id change makes sense. 
But what about the differences in stack frames that I observed on different systems?

Reply:
On Go 1.15 the topmost frame on the original goroutine will always be `runtime.debugCallWrap`, including on darwin, which didn't report an error before because on travis function calls weren't tested.

Reply:
Ah, got it. That must be the frame for launching the separate call goroutine. I added that clause because I observed some test failures on Travis (everything was working on my Mac locally), but with your fix and explanation this makes way more sense now!